buildpack.toml

```diff
@@ -0,0 +1,88 @@
+api = "0.8"
+
+[buildpack]
+  id = "paketo-buildpacks/dotnet-core-aspnet-runtime"
+  homepage = "https://github.com/paketo-buildpacks/dotnet-core-aspnet-runtime"
+  keywords = ["dotnet", "ASP.NET", "runtime"]
+  name = "Paketo Buildpack for ASP.NET Core Runtime"
+  sbom-formats = ["application/vnd.cyclonedx+json", "application/spdx+json", "application/vnd.syft+json"]
+
+  [[buildpack.licenses]]
+    type = "Apache-2.0"
+    uri = "https://github.com/paketo-buildpacks/dotnet-core-aspnet-runtime/blob/main/LICENSE"
+
+[metadata]
+  include-files = ["bin/run","bin/build","bin/detect","buildpack.toml"]
+  pre-package = "./scripts/build.sh"
+
+  [[metadata.dependencies]]
+    cpe = "cpe:2.3:a:microsoft:asp.net_core:3.1:*:*:*:*:*:*:*"
+    deprecation_date = "2024-11-08T00:00:00Z"
+    id = "dotnet-core-aspnet-runtime"
+    name = "ASP.NET Core Runtime"
+    licenses = ["MIT", "MIT-0"]
+    purl = "pkg:generic/dotnet-aspnetcore@3.1.28?checksum=a3fe5848d88dc67cbfceaf0b8e6d3e1178f8441bfedbb21861b2e72fdee3f28d&download_url=https://download.visualstudio.microsoft.com/download/pr/effaa5bf-0fa7-4e5a-9ce8-9ac04ee86669/5afb2b1c2ad68550cec914d8fb303d20/aspnetcore-runtime-3.1.28-linux-x64.tar.gz"
+    sha256 = "a3fe5848d88dc67cbfceaf0b8e6d3e1178f8441bfedbb21861b2e72fdee3f28d"
```

Comment:
should we switch to using `checksum` + the published SHA512 now that it's possible in jam and packit?

Reply:
or do you want to defer that to a subsequent PR?

Reply:
I can take a quick crack at it.